src/AppLayout.tsx

```diff
@@ -1,35 +1,42 @@
-import { Box, Skeleton, styled } from '@mui/material';
+import { Box, useMediaQuery, useTheme } from '@mui/material';
 import Topbar from 'components/header/Topbar';
-import { Suspense, useState } from 'react';
+import { useState } from 'react';
 import { Outlet } from 'react-router';
 import Navigation from './components/navigation/Navigation';
 
 export enum Widths {
+    MOBILE = 0,
     RAIL = 63,
     FULL = 225,
 }
 
 function AppLayout() {
+    const theme = useTheme();
+    const isBelowMd = useMediaQuery(theme.breakpoints.down('md'));
+
     const [navigationOpen, setNavigationOpen] = useState(false);
     const [navWidth, setNavigationWidth] = useState<Widths>(Widths.RAIL);
 
     const toggleNavigationDrawer = () => {
-        setNavigationWidth(navigationOpen ? Widths.RAIL : Widths.FULL);
+        if (!isBelowMd) {
+            setNavigationWidth(navigationOpen ? Widths.RAIL : Widths.FULL);
+        }
```

Comment:
We can skip for smaller screens because they don't have a rail to set the width to.